lib/zipkin-tracer/version.rb

```diff
@@ -1,3 +1,3 @@
 module ZipkinTracer
-  VERSION = '0.25.0'.freeze
+  VERSION = '0.27.0'.freeze
```

Comment:
last time the version didn't get a bump? 🙈
if we just skip 0.26 it's going to look weird in the gem release history.
maybe we should have a micro-PR bumping the number, release 0.26 and then release 0.27, what do you think?